src/task-definitions.js

```diff
@@ -0,0 +1,347 @@
+        dispatcher.addCell('javascript')
+        dispatcher.selectCell(getCellBelowSelectedId(), true)
+      }
+    }
+  },
+})
+
+TASKS.MoveCellUp = new NotebookTask({
+  title: 'Move Cell Up',
+  displayKeybinding: '\u21E7 \u2191',
+  keybindings: ['shift+up'],
+
+  callback() {
+    this.props.actions.cellUp()
+  },
+  keybindingCallback(e) {
+    if (isCommandMode()) {
+      if (e.preventDefault) {
+        e.preventDefault()
+      }
+      dispatcher.cellUp()
+    }
+  },
+})
+
+
+TASKS.MoveCellDown = new NotebookTask({
+  title: 'Move Cell Down',
+  displayKeybinding: '\u21E7 \u2193',
+  keybindings: ['shift+down'],
+
+  callback() {
+    this.props.actions.changeCellType('javascript')
+  },
+  keybindingCallback(e) {
+    if (isCommandMode()) {
+      if (e.preventDefault) {
+        e.preventDefault()
+      }
+      dispatcher.cellDown()
+    }
+  },
+})
+
+
+TASKS.SelectUp = new NotebookTask({
+  title: 'Select Cell Above',
+  displayKeybinding: '\u2191',
+  keybindings: ['up'],
+
+  keybindingCallback(e) {
+    if (isCommandMode()) {
+      // e.preventDefault blocks kbd scrolling of entire window
+      if (e.preventDefault) {
+        e.preventDefault()
+      } else { // internet explorer
+        e.returnValue = false
+      }
+      const cellAboveId = getCellAboveSelectedId()
+      if (cellAboveId !== null) { dispatcher.selectCell(cellAboveId, true) }
+    }
+  },
+})
+
+TASKS.SelectDown = new NotebookTask({
+  title: 'Select Cell Down',
+  displayKeybinding: '\u2193',
+  keybindings: ['down'],
+
+  keybindingCallback(e) {
+    if (isCommandMode()) {
+      // e.preventDefault blocks kbd scrolling of entire window
+      if (e.preventDefault) {
+        e.preventDefault()
+      } else { // internet explorer
+        e.returnValue = false
+      }
+      const cellBelowId = getCellBelowSelectedId()
+      if (cellBelowId !== null) { dispatcher.selectCell(cellBelowId, true) }
+    }
+  },
+})
+
+
+TASKS.AddCellAbove = new NotebookTask({
+  title: 'Add Cell Above',
+  keybindings: ['a'],
+  displayKeybinding: 'a',
+
+  callback() {
+    dispatcher.insertCell('javascript', 'above')
+    dispatcher.selectCell(getCellAboveSelectedId(), true)
+  },
+  keybindingCallback() {
+    if (isCommandMode()) {
+      dispatcher.insertCell('javascript', 'above')
+      dispatcher.selectCell(getCellAboveSelectedId(), true)
+    }
+  },
+})
+
+TASKS.AddCellBelow = new NotebookTask({
+  title: 'Add Cell Below',
+  keybindings: ['b'],
+  displayKeybinding: 'b',
+
+  callback() {
+    dispatcher.insertCell('javascript', 'below')
+    dispatcher.selectCell(getCellBelowSelectedId(), true)
+  },
+  keybindingCallback() {
+    if (isCommandMode()) {
+      dispatcher.insertCell('javascript', 'below')
+      dispatcher.selectCell(getCellBelowSelectedId(), true)
+    }
+  },
+})
+
+TASKS.DeleteCell = new NotebookTask({
+  title: 'Delete Cell',
+  keybindings: ['shift+del', 'shift+backspace'],
+  displayKeybinding: '\u21E7 \u232b',
```

Comment:
if we are going to use unicode, probably we should just use unicode literals rather than codepoints -- but same thought here as for "enter" below-- might be better to use text; i think it's probably clearer across platforms etc

Reply:
Yep, these are holdovers copy-pasted from the menus. Much of this depends on how they are rendered in the `material-ui` components. @mdboom's comment below about acutally just having a fcn that makes a display keybinding is smart (I had that thought while doing this but figured I'd keep them separate for now since I wasn't sure how to deal with multiple keybindings). I think we should have a function that makes the display keybindings from this, and that allows us to deal with formatting stuff without having to be so explicit.